src/Login/Login.tsx

```diff
@@ -0,0 +1,36 @@
+import { FC, useEffect } from "react";
+import classes from './Login.module.scss'
+import { Link, Navigate } from "react-router-dom";
+import { useContext } from "react";
+import { AppContext } from "../Context";
+
+export interface LoginProps {
+
+}
+
+
+export const Login: FC<LoginProps> = () => {
+
+  const context = useContext(AppContext);
+
+  useEffect(() => {
+    document.title = 'Grambotica Login';
+  })
+
+  const clickHandler = () => {
+    context.login = true;
+    context.loginTime = new Date();
+  }
+
+  return (
+    <div className={`${classes['page-container']}`}>
```

Comment:
зачем строка? Почему не так?
```suggestion
    <div className={classes['page-container']}>
```